extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -154,6 +159,11 @@ class AmpApesterMedia extends AMP.BaseElement {
       renderer: true,
       tags: extractTags(this.getAmpDoc(), this.element),
     };
+    const ampdoc = this.getAmpDoc();
+    Services.extensionsFor(this.win)./*OK*/ installExtensionForDoc(
+      ampdoc,
+      AD_TAG
+    );
```

Comment:
Let's not install `amp-ad` until we know there is a bottom ad enabled (around line 334)

Reply:
+1

Reply:
Agree, fixed